invenio_drafts_resources/records/api.py

```diff
@@ -90,6 +93,14 @@ class Record(RecordBase):
     #: Version relationship
     versions = VersionsField(create=True, set_latest=True)
 
+    @property
+    def versions_count(self):
+        """Get number of record's active versions"""
+        records = list(
+            self.get_record_versions(parent_id=self.parent.id, include_deleted=False)
```

Comment:
Could we reuse `get_records_by_parent` with `with_deleted=False, ids_only=True`?

This would fix the tests which are failing due to the import from `invenio_rdm_records` which we cannot do in `invenio-drafts-resources`.
Not sure what is the difference between `is_deleted` and `deletion_status` in the model.

Reply:
After checking, I can confirm that we need to use the `deletion_status` and not `is_deleted=False` as `is_deleted` is checking for empty json (see [here](https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/models.py#L106))

You are right tho, that we can not import `invenio_rdm_records`... I could hardcode the value `"P"` of the `RecordDeletionStatusEnum.PUBLISHED`..